File: python/beginner/askforname.py
#!/usr/bin/python3
import os

print("Hey there! What's your name?")
name = input()
if name == "":
	print("You can not have an empty name...")
else:
	print("Hello " + name + "! Nice to meet you!")

if os.name == "nt":
	input("Press any key to end...")
else:
	exit()

File: c/beginner/ask-for-name-bob-only/askforname.c
#include<stdio.h>
#include<string.h>

int main(void)
{
	char name[256];
	puts("Hey! What's your name?");
	scanf("%s", name);
	if(strcmp(name, "Bob") == 0 || strcmp(name, "Alice") == 0)
	{
		printf("Hello %s! Nice to meet you!\n", name);
	}
	else
	{
		puts("Nothing for you...");
	}
	return 0;
}
